doc/docs.md

```diff
@@ -20,15 +20,17 @@ The major way to get the latest and greatest V, is to __install it from source__
 It is __easy__, and it usually takes __only a few seconds__.
 
 ### Linux, macOS, FreeBSD, etc:
-You need `git`, and a C compiler like `tcc`, `gcc` or `clang`, and `make`:
+You need [`git`](https://git-scm.com/downloads)
+>You can install and use a C compiler of your choice for binary optimization, but by default V will use [TCC](http://savannah.nongnu.org/projects/tinycc/).
```

Comment:
`... by default V will use ...` -> `... by default V will download and use ...` (so that any deployment is clear)

Reply:
ahh yes true true